README.md

```diff
@@ -1,3 +1,6 @@
+[![mops](https://oknww-riaaa-aaaam-qaf6a-cai.raw.ic0.app/badge/mops/bitcoin)](https://mops.one/bitcoin)
```

Comment:
This currently shows a "mops 0.1.1" badge. I guess the idea is that first a release is made on Github, and only afterwards it is pushed to mops, so that the mops release is never ahead of the latest release on Github.